uber/models.py

```diff
@@ -745,7 +745,7 @@ def assign_badges(self, group, new_badge_count, new_badge_type=c.ATTENDEE_BADGE,
 
             if int(new_badge_type) in c.PREASSIGNED_BADGE_TYPES and c.AFTER_PRINTED_BADGE_DEADLINE and diff > 0:
                 return 'Custom badges have already been ordered, so you will need to select a different badge type'
-            elif diff > 0:
+            elif diff > 0 and group.cost != 0:
```

Comment:
:(

RIght now in Prime, we unset auto_recalc on all groups because otherwise we'd constantly be asking for more money as the badge price raised based on bucket pricing. That is why we can't do the more-correct `and not group.auto_recalc`